components/Pagination/index.js

```diff
@@ -91,7 +100,7 @@ class Pagination extends React.Component {
     var elements = [];
 
     forEach(pager.pages(), function(pageNumber) {
-      var className = pageNumber === pager.currentPage ? 'active' : '';
+      var className = pageNumber === pager.currentPage ? 'active' : null;
```

Comment:
about this change, it should have go into a specific commit, setting the class to `''` adds the `class=` attribute while null does not